app/Http/Controllers/CampaignsController.php

```diff
@@ -67,9 +67,14 @@ public function show($campaignId)
             });
         });
 
+        // Get the campaign data
+        $campaign_data = $this->campaignService->find($campaignId);
+
         return view('pages.campaign_inbox')
             ->with('state', [
                 'signups' => $signups,
+                'campaign' => $campaign_data,
+                'title' => $campaign_data['title'],
```

Comment:
I feel like title and could be in the `campaign` property and doesn't need to be separate.